internify/src/components/pages/CreateJobPosting.js

```diff
@@ -0,0 +1,39 @@
+import InputFormJobHeader from '../molecules/InputFormJobHeader';
+import LengthSubHeader from '../molecules/LengthSubHeader';
+import PositionSubHeader from '../molecules/PositionSubHeader';
+import Container from '@material-ui/core/Container';
+import ContinueBtn from '../atoms/ContinueBtn';
+import Stepper from '../molecules/Stepper';
+
+import './styles/CreateJobPosting.css';
+
+function CreateJobPosting(props){
+    return(
+        <div className="CreateJobPosting">
+            <Container maxWidth="md" className={"container"}>
+                <div className="stepper-row">
+                    <Stepper stepNumber={0}/>
```

Comment:
Note: use of passing in props, this will highlight the stepper number. Doing it this way instead of doing something complicated with the state each time we navigate to another page bc this is static on each page. So for each page created, use the Stepper molecule and pass in the stepper icon you want highlighted.

Reply:
Very nice